Sources/MapboxMaps/Location/Puck/Puck2D.swift

```diff
@@ -174,11 +174,85 @@ private extension Puck2DConfiguration {
         topImage ?? UIImage(named: "location-dot-inner", in: .mapboxMaps, compatibleWith: nil)!
     }
 
-    var resolvedBearingImage: UIImage {
-        bearingImage ?? UIImage(named: "location-dot-outer", in: .mapboxMaps, compatibleWith: nil)!
+    var resolvedShadowImage: UIImage {
+        shadowImage ?? UIImage(named: "location-dot-outer", in: .mapboxMaps, compatibleWith: nil)!
     }
 
     var resolvedScale: Value<Double> {
         scale ?? .constant(1.0)
     }
 }
+
+public extension Puck2DConfiguration {
+    // Create a Puck2DConfiguration instance with or without an arrow bearing image. Default without the arrow bearing image.
+    static func createDefault2DPuck(withBearing: Bool = false) -> Puck2DConfiguration {
+        if withBearing {
+            return Puck2DConfiguration(topImage: UIImage(named: "location-dot-inner", in: .mapboxMaps, compatibleWith: nil)!,
+                                    bearingImage: imageArc(),
+                                    shadowImage: UIImage(named: "location-dot-outer", in: .mapboxMaps, compatibleWith: nil)!)
+        } else {
+            return Puck2DConfiguration(topImage: UIImage(named: "location-dot-inner", in: .mapboxMaps, compatibleWith: nil)!,
+                                       bearingImage: UIImage(named: "location-dot-outer", in: .mapboxMaps, compatibleWith: nil)!)
+        }
```

Comment:
Since the only difference is whether the bearingImage is nil or not, refactor this to be less repetitive using the ternary operator.